HomeWork_09/HW9.py

```diff
@@ -0,0 +1,22 @@
+
+text = int(input("Enter a digit: "))
+def my_generator():
+    f1, f2 = 0, 1
+    for my_generator in range(text):
+        yield
+        print(f1,end=" ")
+        f1, f2 = f2, f1 + f2
+
+
+gen = my_generator()
+print(next(gen))
```

Comment:
You should be able to print only an element entered by the user. To make it you can put your generator in a loop, print value that you need and exit the loop. Something like this:
```
n = 0
for value in gen:
    if n == int(text):
        print(value)
        break